doc/encoding.rdoc

```diff
@@ -233,9 +233,12 @@ The default external encoding is:
 The default external encoding is returned by method Encoding.default_external,
 and may be set by:
 
-- \Method Encoding.default_external=.
 - Ruby command-line options <tt>--external_encoding</tt> or <tt>-E</tt>.
```

Comment:
I think it's a bit odd to have a bullet list with only one element. Maybe it should be merged with the sentence above? Something like

```
The default external encoding is returned by method Encoding.default_external,
and may be set through the command-line options <tt>--external_encoding</tt>
or <tt>-E</tt>.
```

Reply:
I wanted to give the item the same weight as those above.

Reply:
That's fair.